src/libraries/Common/src/System/IO/Archiving.Utils.Windows.cs

```diff
@@ -1,17 +1,16 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
+// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-
 using System.Text;
 
-namespace System.IO.Compression
+namespace System.IO
 {
-    public static partial class ZipFileExtensions
+    internal static partial class ArchivingUtils
     {
-        internal static string SanitizeZipFilePath(string zipPath)
+        internal static string SanitizeEntryFilePath(string entryPath)
         {
-            StringBuilder builder = new StringBuilder(zipPath);
-            for (int i = 0; i < zipPath.Length; i++)
+            StringBuilder builder = new StringBuilder(entryPath);
+            for (int i = 0; i < entryPath.Length; i++)
```

Comment:
(There should also be a comment about why these are invalid.  Don't we typically leave such decisions up to the OS now?)

Reply:
This code was originally merged in another PR. I merely renamed the class so it would make sense for usage in both zip and tar.

https://github.com/dotnet/runtime/pull/67332

Since it isn't really new code, I would like to address this suggestion in a future PR.